### PR TITLE
Can not set FACTORIO_RCON_PORT & FACTORIO_RCON_PASSWORD

### PR DESCRIPTION
**Pull request by adamcharnock**
It seems that setting ``FACTORIO_RCON_PORT`` & ``FACTORIO_RCON_PASSWORD`` actually causes the RCON options to **not** be passed to the server at all. I think this is due to a logic error which this PR addresses.

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -32,23 +32,21 @@ echo "###"
...
-if [ -z $FACTORIO_RCON_PORT ]
-then
-  factorio_command="$factorio_command --rcon-port ${FACTORIO_RCON_PORT:-27015}"
-  echo "###"
-  echo "# RCON port is '${FACTORIO_RCON_PORT}'"
-  echo "###"
-fi
+factorio_command="$factorio_command --rcon-port ${FACTORIO_RCON_PORT:-27015}"
+echo "###"
+echo "# RCON port is '${FACTORIO_RCON_PORT}'"
```
Comment (gtaylor) [inline entrypoint.sh]: This can be undefined as-is. You'll want to do something like line 29 in breaking that variable out.

**Review by gtaylor — CHANGES_REQUESTED** [review]
One minor tweak. Thanks!

**Comment by adamcharnock** [comment]
Ah yes, thanks @gtaylor. That should be fixed now.

**Comment by adamcharnock** [comment]
Just fixed another error with this

**Comment by gtaylor** [comment]
Wow, I am not sure how this got list. Sorry about that! A fix got merged in as part of #15.

**Pull request closed without merging**